Userland/Libraries/LibWeb/CSS/Parser/Parser.cpp

```diff
@@ -3056,6 +3070,8 @@ RefPtr<CSSStyleValue> Parser::parse_hsl_color_value(TokenStream<ComponentValue>&
             inner_tokens.discard_whitespace();
 
             alpha = parse_number_percentage_value(inner_tokens);
+            if (!alpha)
+                return {};
```

Comment:
I got a bit confused by this at first, but nice catch! Could you move this fix into its own commit? It's unrelated to the other changes, and that would let you add a commit description for it.

Reply:
Sure, I'll also add a comment then!

Reply:
Moved to another commit!